easybuild/tools/options.py

```diff
@@ -350,6 +350,7 @@ def override_options(self):
             'mpi-cmd-template': ("Template for MPI commands (template keys: %(nr_ranks)s, %(cmd)s)",
                                  None, 'store', None),
             'mpi-tests': ("Run MPI tests (when relevant)", None, 'store_true', True),
+            # optarch is in most cases equivalent to strlist, but changing the type breaks compatibility
```

Comment:
hmm, I think this comment would be more confusing in the long run, it's better to mention this where we split the `optarch` value by `,`?